phase2/src/phase2.rs

```diff
@@ -0,0 +1,95 @@
+use itertools::Itertools;
+
+use std::fs::File;
+use std::fs::OpenOptions;
+
+use parameters::*;
+use utils::Phase2Error;
+
+use crate::utils::map_io_phase2_error;
+
+pub fn contribute(
```

Comment:
Please add method doc.

Reply:
Added in [new commit](https://github.com/MystenLabs/phase2-bn254/pull/1/commits/2174ef8ad275c5996a7dfd2ba93f409cb85ecc08#diff-5159906df3e74880b7c76cd21b4465665f1056e4c86b7bbc42cbeebb01ea7863R11-R13).